hr_employee_age/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2015 Salton Massally (<smassally@idtlabs.sl>).
```

Comment:
`__init__` fields should go w/o Copyright notice as per https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py